### PR TITLE
chore: retract v0.27.7 release

### PR DESCRIPTION
**Pull request by jmdeal**
Fixes #N/A 

**Description**
Retracts the v0.27.7 release. It was accidentally created without patching all of the known vulnerabilities. 

**How was this change tested?**
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by engedaam — DISMISSED** [review]
LGTM 🚀

File: go.mod [hunk 1]
```diff
@@ -105,3 +105,5 @@ require (
 )
 
 retract v0.100.101-test // accidentally published testing version
+
```
Comment (jonathan-innis) [inline go.mod:108]: nit: group retract directives. Any reason we need a newline here?

Comment (jmdeal) [reply go.mod:108]: One of the linters / formatters did it, it failed `make presubmit` without it. I can go ahead and make it a retract block though, I imagine that's fine.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**